### PR TITLE
Small fixes for select org + confirm access splash

### PR DESCRIPTION
**Pull request by wfortin**

### CODE REVIEW

File: scss/elements/buttons-alternate.scss [hunk 1]
```diff
@@ -9,16 +9,21 @@ $btn-login-padding-border: 1px solid $medium-grey;
   width: 240px;
   height: $btn-login-height;
   padding-right: $btn-login-padding-left-right;
-  padding-left: $btn-login-height + $btn-login-padding-left-right;
+  padding-left: $btn-login-padding-left-right;
```
Comment (marilynechan) [inline scss/elements/buttons-alternate.scss]: You should have renamed the `btn-login-x` variables when you renamed to btn-alternate.

File: scss/elements/buttons-alternate.scss [hunk 2]
```diff
@@ -9,16 +9,21 @@ $btn-login-padding-border: 1px solid $medium-grey;
...
+  line-height: $btn-login-height;
 
   background-color: $white;
   border: $btn-login-padding-border;
 
+  &.mod-prepend {
+    padding-left: $btn-login-height + $btn-login-padding-left-right;
+  }
+
   .btn-prepend {
```
Comment (marilynechan) [inline scss/elements/buttons-alternate.scss]: Can you explain mod-prepend vs btn-prepend?

**Comment by JPLachance** [comment]
Look's good to me!

File: scss/elements/buttons-alternate.scss (hunk 2 above)
Comment (wfortin) [inline scss/elements/buttons-alternate.scss]: the mod-prepend is the changing of padding and the btn-prepend is the box where the icon is displayed. Bootstrap calls that the `.add-on` in 2.3

**Pull request merged**